lib/jnpr/junos/utils/sw.py

```diff
@@ -1131,8 +1131,8 @@ def _system_operation(
             elif self._mixed_VC is True:
                 cmd.append(E("all-members"))
         elif (
-            self._multi_VC_nsync is True
-            or self._multi_VC is True
+            (self._multi_VC_nsync is True
```

Comment:
Kindly fix the format for your fix by running "black" tools.  Workflow is failing with your fix

Reply:
Ok, format is now updated according to 'black'.